hms-app/src/components/HackathonDetails.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import { getHackathonDetails } from '../actions/GetBackendData'
+import IdeaCardList from './IdeaCardList'
+
+type IProps = {
+  hackathonID: string
+}
+
+export default function HackathonDetails(props: IProps) {
+  const { hackathonID } = props
+  const [hackathonData, setHackathonData] = useState({
+    errorhackathonData: false,
+    isLoadinghackathonData: true,
```

Comment:
You would need to fix this everywhere they are used, but the names should be proper camel-case:

```suggestion
    errorHackathonData: false,
    isLoadingHackathonData: true,
```